vaadin-grid-pro-flow/src/main/java/com/vaadin/flow/component/gridpro/EditColumnConfigurator.java

```diff
@@ -61,6 +66,16 @@ public EditColumnConfigurator(EditColumn<T> column) {
         return getColumn();
     }
 
+    private <V> Column<T> configureColumn(ItemUpdater<T, String> itemUpdater,
+                                      EditorType type, List<String> options, HasValueAndElement<?, V> editorField) {
```

Comment:
Why do you have `options` as a parameter if the only place where you use this method passes an empty list?

Reply:
Updated.